SubscribeIT/ngDesk
# 273 dashboard save bug

Closes #273 

Test Case 1

Name: Dashboard save bug

1. Login to ngdesk.
2. Goto -> Dashboard -> Add a new dashboard +
3. Create a new dashboard and hit save.
4. Delete the created dashboard.
5. Observe that after deleting the dashboard, its returning to the 'Default (admin)' dashboard.
6. Observe no error.

![Screenshot from 2021-11-15 15-23-17](https://user-images.githubusercontent.com/89504233/141762716-95c29c93-2c3b-4148-acaa-0ab68b342a4a.png)
![Screenshot from 2021-11-15 15-23-24](https://user-images.githubusercontent.com/89504233/141762759-167d8287-6bff-4916-976e-ef7f21bc4117.png)
